packages/itwin/tree-widget/src/components/search-bar/SearchBox.scss

```diff
@@ -50,38 +49,38 @@
   .searchbox-stepping-container {
     display: flex;
     align-items: center;
-    right: 32px;
+    right: var(--iui-size-xl);
     height: 100%;
 
     .searchbox-stepping-count {
-      font-size: 12px;
-      color: $buic-text-color;
-      margin-right: 4px;
+      font-size: var(--iui-size-s);
+      color: var(--iui-color-text);
+      margin-right: var(--iui-size-2xs);
       position: relative;
       word-wrap: none;
     }
 
     .searchbox-separator {
       height: 75%;
       width: 1px;
-      margin: 0 8px;
-      background: $buic-inputs-border;
+      margin: 0 var(--iui-size-xs);
+      background: hsl(var(--iui-color-foreground-hsl) / var(--iui-opacity-4));
     }
   }
 
   .searchbox-step-button {
```

Comment:
Is this class necessary after switching to ItwinUI `IconButton`?

Reply:
Well, this class adds a separator to the search box: 
![image](https://user-images.githubusercontent.com/124659623/227895711-9461ae89-ba31-47b5-9511-a805bb1b73d5.png)

without this class search box would look like this:
![image](https://user-images.githubusercontent.com/124659623/227895805-db7833ed-0d68-4e2e-a508-0dfce626d7fe.png)

I think the we should keep this class